src/Infiltrator.jl

```diff
@@ -428,6 +428,7 @@ All assignments will end up in the `safehouse`.
 The following commands are special cased:
   - `?`: Print this help text.
   - `@trace`: Print the current stack trace.
+  - `@limited_trace`: Print the current stack trace with reduced type information. (Only for Julia >= 1.10)
```

Comment:
I think we should make the reduced info the default and add `@trace_all` if someone wants to print complete type info. Thoughts?

Reply:
I agree, (that's how I would have done it, but I didn't want to change the existing behavior).

I'll update the PR with this

Reply:
I applied the changes and successfully regenerated and re-run the tests.